src/amo/components/Search/index.js

```diff
@@ -26,12 +28,12 @@ export class SearchBase extends React.Component {
     count: PropTypes.number,
     dispatch: PropTypes.func.isRequired,
     enableSearchSort: PropTypes.bool,
+    errorHandler: PropTypes.object.isRequired,
     filters: PropTypes.object,
-    hasSearchParams: PropTypes.bool.isRequired,
+    filtersFromState: PropTypes.object,
```

Comment:
could this be `filtersForResults`? I think that would make it more obvious that it links directly to the `results` property. Saying it comes from `state` is a bit vague since pretty much everything comes from state (even `filters` will come from state once we switch to redux-router).